_posts/2020-06-25-new-release-2020-3(Beta).md

```diff
@@ -0,0 +1,21 @@
+---
+title:  "Tableau 2020.3 Beta released"
+abstract: "Sneak peek on the features available on 2020.3"
+---
+
+With the release of Tableau 2020.3 Beta, you get a sneak peek into the features available in Tableau 2020.3 and connector SDK. 
```

Comment:
Maybe have a line about how certain features can be seen in the 2020.3 branch, but they are not final and may have issues, and that we will merge them into master when 2020.3 is released to the public.